rust/src/treefile.rs

```diff
@@ -911,6 +911,25 @@ impl Treefile {
             .unwrap_or_default()
     }
 
+    pub(crate) fn get_packages_override_replace(&self) -> Vec<crate::ffi::OverrideReplacement> {
+        self.parsed
+            .derive
+            .override_replace
+            .iter()
+            .flatten()
+            .map(|r| {
+                let packages: Vec<String> = r.packages.iter().cloned().collect();
+                match r.from {
+                    RemoteOverrideReplaceFrom::Repo(ref repo) => crate::ffi::OverrideReplacement {
```

Comment:
Let's make this an `impl Into<crate::ffi::OverrideReplacement> for RemoteOverrideReplaceFrom`

Reply:
Good idea!